examples/sycl/pvc/pvc_gemm.cpp

```diff
@@ -365,6 +350,39 @@ int main(int argc, const char** argv)
   ExampleRunner<Gemm> runner;
 
   runner.run(options, hw_info);
+}
+
+int main(int argc, const char** argv)
+{
+  //
+  // Parse options
+  //
+
+  Options options;
+
+  options.parse(argc, argv);
+
+  if (options.help) {
+    options.print_usage(std::cout) << std::endl;
+    return 0;
+  }
+
+  if (options.error) {
+    std::cerr << "Aborting execution." << std::endl;
+    return -1;
+  }
+
+  // row major A, row major B
+  gemm_run<true, true, bfloat16_t, bfloat16_t, float>(options);
+
+  // row major A, column major B
+  gemm_run<true, false, bfloat16_t, bfloat16_t, float>(options);
+
+  // column major A, row major B
+  gemm_run<false, true, bfloat16_t, bfloat16_t, float>(options);
+
+  // column major A, column major B
+  gemm_run<false, false, bfloat16_t, bfloat16_t, float>(options);
```

Comment:
Instead of modifying the example to cover all the cases, we should add unit tests to ensure correctness and benchmark tests to measure performance, leaving the example as a simple demonstration.